salt/modules/cp.py

```diff
@@ -764,15 +777,16 @@ def push_dir(path, glob=None, upload_path=None):
         return push(path, upload_path=upload_path)
     else:
         filelist = []
-        for root, dirs, files in os.walk(path):
+        for root, files in os.walk(path):
```

Comment:
`os.walk` should be returning a three-item tuple. I'm not sure I follow the logic of removing the second assignment here. Could you clarify, please?